Restrict Time units (#41)
ices-publications/SONAR-netCDF4: docs/crr341.adoc

All relevant variables and attributes in SONAR-netCDF4 files are required to have a textual netCDF4 attribute with the name “units” that specifies the units. The International System of Units (SI) is used. For simplicity, the data format mandates the use of particular units and their textual form, as per the definitions and conventions of the UDUNITS-2 package cite:[ucar2014]. Decibels are commonly used in underwater acoustic quantities, but UDUNITS does not currently include such a unit. However, as the CF convention does permit decibel (with symbols: “dB”, “db”, and “dbel”), it is used in the SONAR-netCDF4 convention.

==== Time units

> Even if netcdf and conventions of the UDUNITS-2 allows to specify any valid time unit, to prevent any extra complication to software that reads the SONAR-netCDF4 files only two time units are supported in this convention.

Reviewer: ```suggestion
Even though NetCDF and the UDUNITS-2 part of the CF conventions allow for the specification any valid time unit, to prevent any extra complication in software that reads the SONAR-netCDF4 files, only two time units are supported in this convention.
```

* nanoseconds since 1601-01-01 00:00:00Z

* nanoseconds since 1970-01-01 00:00:00Z

> The choice of nanoseconds is due to the sonar system timestamp precision that can be less than one millisecond. Since other forms of units like 100 nanoseconds are not well supported by the various tools that read CF-formatted time units, nanoseconds units is retained.

Reviewer: ```suggestion
The choice of nanoseconds is because sonar system timestamp precision can be less than one millisecond. Other forms of time units, such as 100 nanoseconds, are not well supported by the various tools that read CF-formatted time units, and nanoseconds units is retained.
```

> Epoch time 1601 is because it is the epoch time unit used in Simrad .raw files but also in Windows32/64  systems.

Reviewer: ```suggestion
The year 1601 is used because it is the epoch year unit used in Simrad .raw files and Windows32/64  systems.
```

> Epoch time 1970 is because it is the epoch time widely used in Unix and most programming langages.

Reviewer: ```suggestion
The year 1970 is used because it is the epoch time widely used in Unix and most programming languages.
```
